packages/peaclock/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/octobanana/peaclock
+TERMUX_PKG_DESCRIPTION="A responsive and customizable clock, timer, and stopwatch for the terminal."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@efreak"
+TERMUX_PKG_VERSION=0.4.3
+TERMUX_PKG_SRCURL=${TERMUX_PKG_HOMEPAGE}/archive/${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
While nice to not have to repeat the url, no other packages does this. It could potentially cause issues for scripts that grep/sed the build.sh script

Reply:
I figured it'd be just as good, since the version number is included in exactly that way in approximately half the packages. Fixed.